samples/bluetooth/broadcast_assistant/README.rst

```diff
@@ -0,0 +1,16 @@
+.. zephyr:code-sample:: bluetooth_broadcast_assistant
+   :name: Bluetooth: Broadcast Assistant
+   :relevant-api: bluetooth
```

Comment:
You'd probably want the sample to show up here https://builds.zephyrproject.io/zephyr/pr/68228/docs/connectivity/bluetooth/api/bap.html (plus, it doesn't look like the `bluetooth` doxygen group is "mounted" anywhere in the documentation anyway) so: 

```suggestion
   :relevant-api: bt_bap
```

Reply:
thanks! - I am sure there will be many things to correct.  I just wanted to push the first draft of the stand-alone assistant when it was in a stable working condition.  (source: a modified broadcast_audio_source sample and the sink: Galaxy Buds2 Pro .. music is flowing.. and it's a nice feeling ;))